Packs/PerplexityAI/Integrations/PerplexityAI/PerplexityAI.py

```diff
@@ -0,0 +1,191 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+import json
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def test_module(self, args: dict):
+        self.chat(args)
+
+    def chat(self, args: dict):
+        try:
+            response = self._http_request(
+                method="POST",
+                url_suffix="/chat/completions",
+                json_data=args
+            )
+        except Exception as e:
+            msg = f"test_module.chat: exception - {e}"
```

Comment:
```suggestion
            msg = f"{demisto.command()}: exception - {e}"
```
The command is not always `test_module` it could be also `deepresearch-chat`